packages/form-js-editor/assets/form-js-editor-base.css

```diff
@@ -404,11 +409,7 @@
 /* row drop preview */
 .fjs-children > .gu-transit {
   height: 28px !important;
-  margin-left: 28px !important;
-  margin-right: -8px !important;
-  flex: unset !important;
```

Comment:
I believe we have to keep `flex: unset`, otherwise it will mess up the height of the previews.

<img width="422" alt="image" src="https://user-images.githubusercontent.com/9433996/229440539-2bd9577c-442b-4e45-b01e-bca2c4a5937c.png">